mozilla-mobile/android-components/components/feature/findinpage/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools">
+
+    <!-- Watermark/Hint for the find in page input field. -->
+    <string name="mozac_feature_findindpage_input">Find in page</string>
+
+    <!-- String to show the number of results found in the page and the
+    position the user is at. The first argument is the position, the second argument is the total. -->
+    <string name="mozac_feature_findindpage_result">%1$d/%2$d</string>
+
+    <!-- String to be read by the accessibility service presenting the number of results found in the page
+    and the position the user is at. The first argument is the position, the second argument is the total. -->
+    <string name="mozac_feature_findindpage_accessibility_result" tools:ignore="PluralsCandidate">%1$d out of %2$d</string>
```

Comment:
Out of curiosity, what's this `tools:ignore="PluralsCandidate"`?

Reply:
`tools:abc` is stuff that Android Studio uses to silence warnings etc. We don't support android plurals in compare-locale nor pontoon, and there's probably a check that says that this should be using plurals.